pom.xml

```diff
@@ -468,11 +475,6 @@
         <version>${google.flogger.version}</version>
         <scope>runtime</scope>
       </dependency>
-      <dependency>
-        <groupId>com.google.api.grpc</groupId>
-        <artifactId>grpc-google-cloud-storage-v2</artifactId>
```

Comment:
this is present in gcs/pom.xml. Is it still safe to remove?

Reply:
Yes. As it is getting added as part of `google-cloud-storage`.